### PR TITLE
changes for native build of ec lib

### PR DESCRIPTION
**Pull request by jan-schreib**
in this branch because of travis

### CODE REVIEW

**Comment by jan-schreib** [comment]
@Qabel/developer travis passes. Would be nice if you provide some feedback. Feel free to modify this by yourself.

File: build.gradle [hunk 1]
```diff
@@ -22,24 +23,123 @@ dependencies {
...
+
+    def builderTask = binary.tasks
+    nativeJar.into("de/qabel/platform/$variantName") { from builderTask.outputFile }
+    nativeJar.dependsOn builderTask
+    test.dependsOn nativeJar
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.2.1'
+    //gradleVersion = '2.2.1'
```
Comment (audax) [inline build.gradle]: Why did you comment this out and didn't remove it instead?

**Comment by audax** [comment]
I think we should not the additional requirements to build qabel-core: The gcc.

**Comment by jan-schreib** [comment]
This README is already outdated, but you are right.

**Comment by jasinai** [comment]
If you update the README, I will merge this PR. :wink:

**Comment by audax** [comment]
First.

…scnr.

**Pull request merged**